spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -22,6 +22,37 @@
 == Release Notes
 This section documents the changes introduced by individual releases.
 
+[[release_notes_4_1]]
```

Comment:
We usually do this before the release, but I don't mind.

Reply:
It is WIP, so I as well think this won't hurt.
On a tangent, we should look into the current [milestone](https://github.com/eclipse/microprofile-health/milestone/8), and finalize it somehow, WDYT?